README.md

```diff
@@ -676,9 +670,7 @@ Clients for commercial social platforms that had their API access cut off in a w
 
 - [Anjuta](https://gitlab.gnome.org/Archive/anjuta) - (archived) IDE with a GUI designer for the GNOME desktop `#c` `#gtk2` `#gnome`.
 - [GNOME Builder](https://apps.gnome.org/Builder) - Tool to help you write and contribute to great GNOME-based applications `#c` `#gtk4` `#libadwaita` `#gnome`.
-- [GtkIDE.jl](https://github.com/jonathanBieler/GtkIDE.jl) - GTK-based IDE for Julia `#julia` `#gtk3`.
```

Comment:
Same here, I don't think this should be removed as there isn't an alternative Julia IDE on the list.

Reply:
Despite Jonathan's support for GTK bindings for julia, it's clear that this project of his is stagnating